rainier-benchmark/benchmarks.txt

```diff
@@ -1,44 +1,45 @@
-run                  BinomialPoissonApproximation 10000                       4000.000
-run                  BinomialPoissonApproximation 100000                     40000.000
-run                  Exponential               100                           30.000
-run                  Exponential               1000                         300.000
-run                  Exponential               10000                       3000.000
-run                  Exponential               100000                     30000.000
-run                  GaussianMixture           100                           50.000
-run                  GaussianMixture           1000                        1000.000
-run                  GaussianMixture           10000                       5000.000
-run                  GaussianMixture           100000                     50000.000
-run                  Geometric                 100                           20.000
-run                  Geometric                 1000                         200.000
-run                  Geometric                 10000                       2000.000
-run                  Geometric                 100000                     20000.000
-run                  GeometricZeroInflated     100                          100.000
-run                  GeometricZeroInflated     1000                        1000.000
-run                  GeometricZeroInflated     10000                      10000.000
-run                  GeometricZeroInflated     100000                    100000.000
-run                  Laplace                   100                           30.000
-run                  Laplace                   1000                         300.000
-run                  Laplace                   10000                       3000.000
-run                  Laplace                   100000                     30000.000
-run                  LogNormal                 100                           30.000
-run                  LogNormal                 1000                         300.000
-run                  LogNormal                 10000                       3000.000
-run                  LogNormal                 100000                     30000.000
-run                  NegativeBinomial          100                           20.000
-run                  NegativeBinomial          1000                         200.000
-run                  NegativeBinomial          10000                       2000.000
-run                  NegativeBinomial          100000                     20000.000
-run                  Normal                    100                           20.000
-run                  Normal                    1000                         300.000
-run                  Normal                    10000                       3000.000
-run                  Normal                    100000                     30000.000
+run                  Bernoulli                 100                            0.300
```

Comment:
This now looks like it did before the optimization-removal in https://github.com/stripe/rainier/pull/427